### PR TITLE
OboeTester: Add checkbox for setStreamControlByAttributes

### LINKED ISSUE #1923: OboeTester: need checkbox to disable call to setStreamControlByAttributes()
This was added in #1886
Just needed in TEST OUTPUT.
Maybe fit it next to "Signal:" menu.

### PR DESCRIPTION
**Pull request by robertwu1**
Fixes #1923

### CODE REVIEW

File: apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestOutputActivity.java [hunk 1]
```diff
@@ -179,6 +184,13 @@ public void onChannelBoxClicked(View view) {
         mAudioOutTester.setChannelEnabled(channelIndex, checkBox.isChecked());
     }
 
+    @Override
+    protected void setStreamControlByAttributes(int usage, int contentType) {
+        if (mShouldSetStreamControlByAttributes.isChecked()) {
+            super.setStreamControlByAttributes(usage, contentType);
```
Comment (philburk) [inline apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestOutputActivity.java:190]: A set() method should set().
We should condition the call to the set() method.
The method that is conditional might be called "maybeSet.....".

**Review by philburk — CHANGES_REQUESTED** [review]

File: apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestOutputActivity.java (hunk 1 above)
Comment (robertwu1) [reply apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestOutputActivity.java]: Thanks! Done!

File: apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestAudioActivity.java [hunk 2]
```diff
@@ -604,7 +604,7 @@ public void openAudio() throws IOException {
      * @param usage usage for the stream
      * @param contentType tupe of the stream
      */
-    protected void setStreamControlByAttributes(int usage, int contentType) {
+    protected void maybeSetStreamControlByAttributes(int usage, int contentType) {
```
Comment (philburk) [inline apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestAudioActivity.java:607]: This is the method that sets it (assuming the API is implemented).

**Review by philburk — CHANGES_REQUESTED** [review]
This is not what I meant.

File: apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestAudioActivity.java (hunk 2 above)
Comment (robertwu1) [reply apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestAudioActivity.java]: See newest iteration. I added a helper function

**Review by philburk — APPROVED** [review]

**Pull request merged**